src/client/components/MyProfile/YourUploadedCVs/CVList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './YourUploadedCVs.css';
```

Comment:
Capital letter is only for components or classes

```suggestion
import './yourUploadedCVs.css';
```